vm/systemSmartContracts/governance.go

```diff
@@ -1010,7 +1034,7 @@ func (g *governanceContract) getTotalStake(validatorAddress []byte) (*big.Int, e
 	return validatorData.TotalStakeValue, nil
 }
 
-func (g *governanceContract) saveUserVotes(address []byte, votedList *OngoingVotedList) error {
+func (g *governanceContract) saveUserVotes(address []byte, votedList *OngoingVotedListV2) error {
 	marshaledData, err := g.marshalizer.Marshal(votedList)
```

Comment:
if not activated you have you marshal as OngoingVotedListV1. Check: func (s *stakingSC) saveStakingData

Reply:
done